src/client/article.ts

```diff
@@ -53,14 +53,17 @@ function insertAds(): void {
 
         const observer = new MutationObserver(callback);
         observer.observe(targetNode, config);
+
+        // handle font loading and changes that do not mutate DOM
+        setTimeout(() => nativeClient.updateAdverts(getAdSlots()), 200);
```

Comment:
Every 200ms forever? If this is for content like fonts that load in relatively early, could we have this run for mebbe the first 5-10 seconds and see if that works ok?

Reply:
This will only run once after 200ms. I think this probably isn't a problem when we're loading fonts from device. Maybe it does need to be a few times after 200ms just to make sure...

The mutation Observer is working well though, clicking to follow an author (changes DOM) will reposition the ads.

Reply:
Ah my bad, sorry, I read that as `setInterval`.

Hmm, yeah I wonder if it's worth running it a handful of times since font load times are non-deterministic. It might be very fast on recent devices, but perhaps a touch slower on older ones?

Reply:
There's the [font loading API](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Font_Loading_API) which might work for this purpose. It looks like it's mostly supported in both Safari and Chrome, although I'm not 100% sure about the font loading event in Safari...